src/collaboration/connection-manager.js

```diff
@@ -149,16 +151,16 @@ module.exports = class ConnectionManager extends EventEmitter {
       if (this._outboundConnections.has(peerInfo)) return
 
       // Check if we're already dialing the peer
-      if (this._dials.has(peerInfo)) return
+      if (this._dials.isDialing(peerInfo)) return
 
       // Dial the peer
-      this._dials.add(peerInfo)
+      const dialId = this._dials.add(peerInfo)
```

Comment:
@dirkmc since the `_dials` are scoped to a collaboration, do we need to track many dials to the same peer?

Reply:
@dirkmc understood, thanks!